code/datums/traits.dm

```diff
@@ -1118,3 +1118,20 @@ obj/trait/pilot
 			var/mob/living/carbon/human/H = owner
 			H.set_mutantrace(/datum/mutantrace/roach)
 		return
+
+/obj/trait/cat
+	name = "Feline (-1) \[Species\]"
+	cleanName = "Cat"
+	icon_state = "placeholder"
```

Comment:
placeholder?

Reply:
Placeholder is a "?" in the same style as the other trait icons. I thought it was more fitting than reusing cateye or making a new icon and failing to replicate the style, and that it complimented the joke.